src/main/java/vg/civcraft/mc/civmodcore/dao/ManagedDatasource.java

```diff
@@ -62,7 +66,8 @@
 	private static final long MAX_WAIT_FOR_LOCK = 600000l;
 	private static final long WAIT_PERIOD = 500l;
 	
-	private ConnectionPool connections;
+	private HikariDataSource connections;
+	private HikariConfig config;
```

Comment:
I think I'll just finish out my thought there, which is to make a serializable configuration type that we can expand as we desire, but yes at is stands its utility is low (unless we wanted to support pool teardown/recreation during runtime at some point, which is what i had in mind in leaving this in there... just didn't finish out the rest of the methods to support it.)